server/journeys/record-investigation/interview-details/controller.ts

```diff
@@ -0,0 +1,69 @@
+import { Request, Response } from 'express'
+import { SchemaType } from './schemas'
+import { BaseJourneyController } from '../../base/controller'
+import { formatInputDate } from '../../../utils/datetimeUtils'
+
+export class InterviewDetailsController extends BaseJourneyController {
+  GET = async (req: Request, res: Response) => {
+    const index = Number(req.params['index']) - 1
+
+    if (Number.isNaN(index) || !req.journeyData.investigation?.interviews?.[index]) {
+      return res.render('pages/error', {
+        message: 'Interview not found',
+        status: 404,
+      })
+    }
+
+    const intervieweeRoleOptions = await this.getReferenceDataOptionsForRadios(
+      req,
+      'interviewee-role',
+      res.locals.formResponses?.['intervieweeRole'] ||
+        req.journeyData.investigation!.interviews![index]!.intervieweeRole,
+    )
+    return res.render('record-investigation/interview-details/view', {
+      interviewText:
+        res.locals.formResponses?.['interviewText'] || req.journeyData.investigation?.interviews![index]?.interviewText,
+      intervieweeRoleOptions,
+      intervieweeRole:
+        res.locals.formResponses?.['intervieweeRole'] ||
+        req.journeyData.investigation?.interviews![index]?.intervieweeRole,
```

Comment:
the pre-selected answer is included in `intervieweeRoleOptions`.
i don't think you need this `intervieweeRole` property?

Reply:
oh yeah good catch!

Reply:
oh yeah, good catch!